plugins/inputs/logfile/logfile.go

```diff
@@ -205,6 +171,13 @@ func (t *LogFile) FindLogSrc() []logs.LogSrc {
 				continue
 			}
 
+			if t.numUsedFds.GetLimit() != 0 {
+				if ok := t.numUsedFds.Acquire(defaultTimeoutToAcquire); !ok {
+					t.Log.Debugf("Cannot increase counter of used file descriptors")
```

Comment:
Nitpick:
Can use `t.Log.Debug()` instead of `t.Log.Debugf()` if you have no format string.